documentation/modules/metrics/proc-metrics-kafka-deploy-options.adoc

```diff
@@ -26,7 +26,7 @@ Perform the following steps for each `Kafka` resource in your deployment.
 +
 [source,shell,subs="+quotes,attributes"]
 ----
-kubectl edit kafka _KAFKA-CONFIG-FILE_
+kubectl edit kafka
```

Comment:
I don't think this is correct. The `edit` command should have a name of the Kafka custom resource which should be edited.

Reply:
I'll revert this.